apps/web/components/Layout.js

```diff
@@ -34,7 +34,7 @@ export const Layout = ({ layoutReady, children }: Props) => {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    overflow: 'hidden',
+    overflow: 'scroll',
```

Comment:
Can we use `auto` instead? Scroll is ok on Mac, but on windows it will make scrollbar visible all the time.